This definition from the header was subsequently removed in 2.27.1 but was left behind
in this header for 2.26.2. This causes an error building +quartz.
see: https://trac.macports.org/ticket/59704#comment:10
kencu@macports.org

--- Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/LayerTreeHost.h.orig	2019-11-19 23:19:08.000000000 -0800
+++ Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/LayerTreeHost.h	2019-11-19 23:19:22.000000000 -0800
@@ -212,7 +212,6 @@
 inline void LayerTreeHost::cancelPendingLayerFlush() { }
 inline void LayerTreeHost::setRootCompositingLayer(WebCore::GraphicsLayer*) { }
 inline void LayerTreeHost::setViewOverlayRootLayer(WebCore::GraphicsLayer*) { }
-inline void LayerTreeHost::invalidate() { }
 inline void LayerTreeHost::scrollNonCompositedContents(const WebCore::IntRect&) { }
 inline void LayerTreeHost::forceRepaint() { }
 inline bool LayerTreeHost::forceRepaintAsync(CallbackID) { return false; }
